core/src/css/ionic/core.ionic.scss

```diff
@@ -8,10 +8,6 @@
   --background: #{globals.$ion-bg-surface-default};
 }
 
-html {
-  --ionic-dynamic-font: -apple-system-body;
```

Comment:
This is no longer needed as it is defined by the base tokens: https://github.com/ionic-team/ionic-framework/blob/3fd1d5cab14a866354d5d21fe15e00661ef0f922/core/src/themes/base/default.tokens.ts#L116